cli/sub_command.go

```diff
@@ -243,6 +308,44 @@ func (c *subCmd) subscribe(p *fisk.ParseContext) error {
 	}
 
 	switch {
+	case c.reportSubjects:
+		sub, err = nc.Subscribe(c.subject, subjectReportHandler)
+
+		go func() {
```

Comment:
might be worth pulling this into a seperate function, this one is becoming (long ago) a monster

Reply:
See: https://github.com/nats-io/natscli/pull/738/commits/aa3535a7678763bcb8998bade29a6a23dfe17a49